pkg/rehearse/jobs.go

```diff
@@ -16,20 +16,23 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/selection"
 	"k8s.io/apimachinery/pkg/util/sets"
+
+	"k8s.io/client-go/rest"
 	"k8s.io/client-go/testing"
+
 	pjapi "k8s.io/test-infra/prow/apis/prowjobs/v1"
 	pjclientset "k8s.io/test-infra/prow/client/clientset/versioned"
 	pjclientsetfake "k8s.io/test-infra/prow/client/clientset/versioned/fake"
 	pj "k8s.io/test-infra/prow/client/clientset/versioned/typed/prowjobs/v1"
-
 	prowconfig "k8s.io/test-infra/prow/config"
 	"k8s.io/test-infra/prow/pjutil"
-
-	"k8s.io/client-go/rest"
 )
 
 const (
 	rehearseLabel = "ci.openshift.org/rehearse"
+
+	KubeAgent    = "kubernetes"
```

Comment:
These should be imported

Reply:
It should be comparable, if not, cast it to a ProwJobType then comapre

Reply:
It's not comparable and I cannot cast it without creating an interface for that, which will lead to a panic. I will keep looking for a better way, maybe I missed something here

Reply:
my bad. It seems that `source.Agent == string(pjapi.KubernetesAgent)` is comparable. No need to cast it to a `ProwJobType`